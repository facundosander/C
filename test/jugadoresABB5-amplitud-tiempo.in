crearArbolJugadores

amplitudArbolJugadoresTiempo 10000 2

liberarArbolJugadores

crearArbolJugadores

amplitudArbolJugadoresTiempo 16383 2

liberarArbolJugadores

crearArbolJugadores

amplitudArbolJugadoresTiempo 16381 2

liberarArbolJugadores

Fin