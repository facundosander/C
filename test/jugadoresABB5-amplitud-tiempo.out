1>2>La amplitud del árbol es: 4096.
Bien.
3>4>5>La amplitud del árbol es: 8192.
Bien.
6>7>8>La amplitud del árbol es: 8190.
Bien.
9>10>Fin.
